region/norfair/east.json

```diff
@@ -1778,7 +1778,7 @@
                     "heatProof",
                     "canHeatRun"
                   ]},
-                  "reset",
+                  {"resetRoomAtNode": [2]},
```

Comment:
Needs reset at opposite side.
Also, traversing from 2 to 1 requires a reset at 2.